pkg/addon/template.go

```diff
@@ -145,7 +150,8 @@ func NewTemplateData(
 				MonitoringEnabled: cluster.Spec.MLA != nil && cluster.Spec.MLA.MonitoringEnabled,
 				LoggingEnabled:    cluster.Spec.MLA != nil && cluster.Spec.MLA.LoggingEnabled,
 			},
-			CSIMigration: csiMigration,
+			CSIMigration:                csiMigration,
+			KubeVirtInfraStorageClasses: kubeVirtStorageClasses,
```

Comment:
I am not sure if we should not add a new struct for cloud provider specifc data.
We already have a field `cloudProviderName`, so we could create 
```
type CloudProviderData struct {
    KubeVirtData // struct ...
}
```

I am not sure because KubeVirt is an exception and maybe other cloud providers do not need this.
Don't change that yet, we need another opinion.